code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -1592,6 +1592,79 @@
 	glass_name = "Meatshake"
 	glass_desc = "Blended meat and cream for those who want crippling health issues down the road. Has two straws for sharing! Perfect for dates!"
 
+/datum/reagent/drink/toothpaste
+	name = "Toothpaste"
+	id = "toothpaste"
+	description = "A paste commonly used in oral hygiene."
+	reagent_state = LIQUID
+	color = "#b1eae8"
+	taste_description = "toothpaste"
+	overdose = REAGENTS_OVERDOSE
+	var/strength = 50
+
+	glass_icon_state = "toothpaste"
+	glass_name = "glass of toothpaste"
+	glass_desc = "Dentists recommend drinking zero glasses a day, and instead brushing normally."
+	glass_center_of_mass = list("x"=7, "y"=8)
+
+/datum/reagent/drink/toothpaste/affect_ingest(var/mob/living/carbon/human/M, var/alien, var/removed)
+
+	if(!istype(M))
+		return
+
+	if(alien == IS_VAURCA)
+		M.intoxication += (strength / 100) * removed * 3.5
+
+/datum/reagent/drink/toothpaste/cold_gate
+	name = "Cold Gate"
+	id = "cold_gate"
+	description = "A C'thur Favorite, guaranteed to make even the bloodiest of warriors mandibles shimmer."
+	strength = 25
+	taste_description = "mint"
+
+	glass_icon_state = "cold_gate"
+	glass_name = "glass of Cold Gate"
+	glass_desc = "A C'thur Favorite, guaranteed to make even the bloodiest of warriors mandibles shimmer."
+	glass_center_of_mass = list("x"=7, "y"=8)
+
+/datum/reagent/drink/toothpaste/waterfresh
+	name = "Waterflesh"
```

Comment:
i don't think it's supposed to be spelt "waterFLESH" instead of water "FRESH"
is this intentional?